crowdin/messages.json

```diff
@@ -1850,6 +1845,7 @@
   "-2109493526": "2. Set purchase conditions",
   "-179851357": "3. Set restart conditions",
   "-1508069714": "4. Run bot",
+  "-625813641": "${current_step} in DBot trading",
```

Comment:
same here

Reply:
The pattern used in this file is wrong, this will be fixed in [this PR](https://github.com/binary-com/deriv-com/pull/3564)
also this file is useless and should be removed: `src/pages/dmt5/_dmt5-video.tsx`